ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -325,6 +376,18 @@ func deleteCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecscl
 
 	return nil
 }
+func deleteEmptyClusterPrompt(reader *bufio.Reader, cliParams *config.CLIParams) error {
+	fmt.Printf("CloudFormation stack not found for cluster '%s'. Are you sure you want to proceed? [y/N]\n", cliParams.Cluster)
```

Comment:
As discussed irl, I think that the user should only have to type yes to confirm once. 

```
$ ecs-cli down
Are you sure you want to delete your cluster? [y/N]
y
CloudFormation stack not found for cluster 'scalefixtest'. Are you sure you want to proceed? [y/N]
y
INFO[0008] Deleted cluster                               cluster=scalefixtest
```
Instead it could print the messages and then ask for a confirmation once. The issue is that both of the prompts are basically asking the same thing- "do you want to delete your cluster?"